frontend/packages/knative-plugin/src/actions/providers.ts

```diff
@@ -143,6 +144,16 @@ export const useChannelActionProvider = (resource: K8sResourceKind) => {
   return [actions, !inFlight, undefined];
 };
 
+const getEvetingEnabledAddAction = () => {
+  const disabledAddActions = getDisabledAddActions();
+  return {
+    isEventSourceAddEnabled: !disabledAddActions?.includes('knative-event-source'),
+    isEventSinkAddEnabled: !disabledAddActions?.includes('knative-event-sink'),
+    isChannelAddEnabled: !disabledAddActions?.includes('knative-eventing-channel'),
+    isBrokerAddEnabled: !disabledAddActions?.includes('knative-eventing-broker'),
+  };
```

Comment:
Instead of using the strings, can we add these 4 into const file and use it? Similar to `const isEventSinkTypeEnabled = isCatalogTypeEnabled(EVENT_SINK_CATALOG_TYPE_ID);`  

`EVENT_SINK_CATALOG_TYPE_ID` is imported from const.ts file